shepherd/Goal.py

```diff
@@ -67,4 +78,50 @@ def apply_powerup(self, effect, alliance):
             effect - enum representing an effect to be applied to the goal
             alliance - the alliance trying to apply the effect to the goal
         """
-        pass
+        if effect == POWERUP_TYPES.TWO_X:
+            if alliance.name == CONSTANTS.BLUE:
+                if self.blue_two_x_timer.is_running():
```

Comment:
I feel like these blocks can be added into a helper function. It might be helpful to create a map from constant to object (e.g. POWERUP_TYPES.TWO_X and CONSTANT.BLUE to self.blue_two_x_timer somehow)

Reply:
Are these timer variables not already kinda a map from a timer to the constants since they take in the constant when initialized?

Reply:
I do agree the blocks are a bit long and could be put into a helper function however for readability. Functions such as "check_apply(effect, alliance, timer)" or something of that kind.